AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/EComponentHolder.java

```diff
@@ -76,6 +77,10 @@ public JBlock getInitBody() {
 		return getInit().body();
 	}
 
+	public JBlock getIntentFilterTarget(IntentFilterData intentFilterData) {
```

Comment:
i decided to go with `getIntentFilterInitializationBlock` as it only initializes the intentFilter.